src/main/java/bob/task/Event.java

```diff
@@ -0,0 +1,83 @@
+package bob.task;
+
+// solution below adapted from partial solution provided in course website
+// https://nus-cs2103-ay2425s2.github.io/website/schedule/week2/project.html under A-Inheritance
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a Task with a start and end time. A <code>Event</code> object corresponds to
+ * a Task represented by its description, its start time and its end time.
+ */
+public class Event extends Task {
+
+    protected LocalDateTime from;
+    protected LocalDateTime to;
+
+    /**
+     * Creates a new instance of an Event task.
+     *
+     * @param description Description of the task.
+     * @param from Start time of the task.
+     * @param to End time of the task.
+     */
+    public Event(String description, LocalDateTime from, LocalDateTime to) {
+        super(description, Task.Type.EVENT);
+        this.from = from;
+        this.to = to;
+    }
+
+    /**
+     * Returns the string representation of the task.
+     * Includes the description, start and end time of the task.
+     *
+     * @return String representation of the task.
+     */
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (from " + this.getFrom() + " to " + this.getTo() + ")";
+    }
+
+    /**
+     * Returns the string representation of the type of task.
+     *
+     * @return Letter representation of the type of task.
+     */
+    public String getType() {
+        return "E";
+    }
```

Comment:
Since this extends from Task, perhaps you may want to add an Override? This comment applies to your other objects where there is no Override notation.

```suggestion
    /**
     * Returns the string representation of the type of task.
     *
     * @return Letter representation of the type of task.
     */
    @Override
    public String getType() {
        return "E";
    }
```

Reply:
Noted, thank you for the suggestion.